### DISCUSSION TITLE
Failed to resolve 'regenerator-runtime' after upgrade to beta.3 from beta.2

### DISCUSSION THREAD

**Discussion by Himavanth** [parcel-bundler/parcel#6339] [Q&A]
I think this is caused by the change of relative path for resolving. https://github.com/parcel-bundler/parcel/pull/6278

We have a CLI module which is used to package and/or serve a UI app. So Parcel would be installed in the CLI and not the UI app itself. I have `regenerator-runtime` in the UI app's `node_modules`. So this was working earlier with beta.2. Now that the internal JS Transformer is used for helpers and `regenerator-runtime`, one **possible solution** seems to be to install it in my CLI module explicitly using `npm i @regenerator-runtime` and then it works fine.

My question is if this is the expected behaviour and if the solution described above is the right one or if there is a way to tell Parcel to resolve it from the UI app's `node_modules` ?

**Comment by mischnic** [comment]
This is a bug that will be fixed by https://github.com/parcel-bundler/parcel/pull/6326

**Reply by Himavanth** [reply]
Awesome. Thx @mischnic
